src/About/about.scss

```diff
@@ -20,7 +20,7 @@ $section-gap: 2ex;
         padding-left: 2ex;
         padding-right: 2ex;
 
-        padding-top: 1.2ex;
+        padding-top: 1.3ex;
```

Comment:
Why did you change this? Since you did not work on the about pane, I assume this was just done by mistake, so please revert it.